### PR TITLE
[wip] add vagrant fixes, although still a few errors

### PR DESCRIPTION
**Pull request by ghost**
- [ ] site-tree broken
- [ ] vagrant deployment/dhcp script broken

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -26,5 +26,5 @@ six>=1.9.0
 sqlparse==0.1.16
 Unidecode==0.04.18
 tablib==0.11.2
-git+https://github.com/idlesign/django-sitetree.git@a626559c39ff1e865cde3441c44f42864c648dfb
+django-sitetree==1.5
```
Comment (mkurek) [inline requirements/base.txt:29]: are you sure it'll work? :) why 1.5, not 1.6?

Comment (ghost) [reply requirements/base.txt:29]: unfortunatelly it doesnt work
So it's issue to resolve, because menu is not generated after this.

**Comment by mkurek** [comment]
Duplicate of #2938

**Comment by vi4m** [comment]
@xliiv there's the difference between this PR and linked one?! Do we really want to close it?

**Comment by ghost** [comment]
I'll fix the npm part in seperate PR. Now I'm working on it.

**Comment by ghost** [comment]
js part fixed:
https://github.com/allegro/ralph/pull/3055

**Pull request closed without merging**